.pack.mk

```diff
@@ -27,10 +27,28 @@ prepare:
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
-		if [ -n "${GIT_TAG}" ]; then \
-			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
+		if [ "${OS}" = "alpine" ]; then \
+			if [ -n "${GIT_TAG}" ]; then \
+				if $$(echo ${GIT_TAG} | grep -q "entrypoint"); then \
+					export VERSION="$$(echo ${GIT_TAG} | sed 's/-entrypoint//')_alpha0_p0"; \
+				else \
+					export VERSION="$$(echo ${GIT_TAG} | sed 's/-/_/')"; \
+				fi; \
+			else \
+				if $$(echo ${GIT_DESCRIBE} | grep -q "entrypoint"); then \
+					# Set VERSION to format X.X.X_alpha0_pXXX \
+					# See for details: https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#pkgver \
+					export VERSION="$$(echo ${GIT_DESCRIBE} | cut -d- -f1)_alpha0_p$$(echo ${GIT_DESCRIBE} | cut -d- -f3)" ; \
+				else \
+					export VERSION="$$(echo ${GIT_DESCRIBE} | sed ${SED_REPLACE_VERSION_REGEX})"; \
```

Comment:
Have you tested this case locally? Obviously, you will get an error about invalid version. Please fix this. Looks like it should be `export VERSION="$$(echo ${GIT_TAG} | sed 's/-/_/')"; \`.

Reply:
fixed and tested for 2.11.0-entrypoint 2.11-beta1 etc and for 2.10.4-1-g02908e625.